pdsClient/pdsClient.go

```diff
@@ -245,6 +245,33 @@ func (c *E3dbPDSClient) ProxyListRecords(ctx context.Context, authToken string,
 	return result, e3dbClients.FlatMapInternalError(internalError)
 }
 
+// BatchGetRecords makes a call to batch get the records in params,
```

Comment:
Not a proxied user call and not an `internal` endpoint, so who's the consumer of this function? Or is it just a testing helper?

Reply:
It tests the imperfect Scala code I wrote to make, follows the development of the feature starting with implementing the batch pds endpoint and then writing this to test it faster before writing the scala integration test. And now, if we ever have need for a client to get a batch of records in go, the code is ready and tested.